tests/phpunit/Mutator/ProfileListProvider.php

```diff
@@ -70,8 +70,8 @@ private function __construct()
 
     public static function mutatorNameAndClassProvider(): Generator
     {
-        foreach (ProfileList::ALL_MUTATORS as $name => $class) {
-            yield [$name, $class];
+        foreach (self::implementedMutatorProvider() as [$filePath, $className, $shortClassName]) {
```

Comment:
This is more reliable as the source of truth for the list of mutators are the ones _found_ not the ones registered in the `ProfileList` which we are testing...